src/contrib/cluster/Akka.DistributedData/ORMultiValueDictionary.cs

```diff
@@ -27,6 +27,16 @@ internal interface IORMultiValueDictionaryKey
         Type ValueType { get; }
     }
 
+    /// <summary>
+    /// INTERNAL API.
+    ///
+    /// For serialization purposes.
+    /// </summary>
+    internal interface IORMultiValueDictionaryDeltaOperation
+    {
+        bool WithValueDeltas { get; }
```

Comment:
This property is not needed - it's a property of the `ORMultiValueDictionary`, not its delta operations.

Reply:
Done